dist/angular-ui-notification.js

```diff
@@ -145,7 +160,7 @@ angular.module('ui-notification').provider('Notification', function() {
 
                 setCssTransitions('none');
 
-                angular.element(document.querySelector(args.container)).append(templateElement);
+                angular.element(document.getElementsByTagName('body')).append(templateElement);
```

Comment:
Why is this now using the body?
What if I want to use the notification inside a modal? I prefer the flexibility of providing my own selector. 

And just FYI, you can get the body using `document.body`